src/imp/platform/browser/options_parser.js

```diff
@@ -1,9 +1,11 @@
+const util = require('./util');
+
 /* global WorkerGlobalScope */
 // Find the HTML element that included the tracing library (if there is one).
 // This relies on the fact that scripts are executed as soon as they are
 // included -- thus 'this' script is the last one in the array at the time
 // this is run.
-let hostScriptElement = (function () {
+function hostScriptElement() {
```

Comment:
On second glance, I think we need to immediately invoke this function like we were previously. The comment explains why this is the case.

Reply:
Done! This means I do have to add the is browser check in the hostscript element function, since it runs first.